compiler/x/codegen/DataSnippet.hpp

```diff
@@ -41,7 +41,8 @@ class X86DataSnippet : public TR::Snippet
    uint8_t*                       getRawData()                             { return _data.data(); }
    virtual size_t                 getDataSize() const                      { return _data.size(); }
    virtual uint32_t               getLength(int32_t estimatedSnippetStart) { return static_cast<uint32_t>(getDataSize()); }
-   virtual bool                   setClassAddress(bool isClassAddress)     { return _isClassAddress = isClassAddress;}
+   bool                           isClassAddress()                         { return _isClassAddress; }
+   bool                           setClassAddress(bool isClassAddress)     { return _isClassAddress = isClassAddress;}
```

Comment:
Any reason for the setter to return a bool?

Reply:
I think that is historical in the TR code base.  Lots of them do that.  I'm not exactly sure why because I don't think I've ever seen any code take advantage of that property.  Want me to fix?

Reply:
Oh I see, sorry I thought that it was a new API; I think it's fine for this PR, maybe we can clean it up once we determine that there's no impact to existing code.

Reply:
FYI:  Maybe time to dust this off: #4575